### ISSUE TITLE
The torch deep learning model should not use softmax last layer + CrossEntropyLoss 

### ISSUE THREAD

**Issue by yxu02** [MicrosoftDocs/ml-basics#48]
Torch CrossEntropyLoss expects logits instead of normalized probabilities.

**Comment by GraemeMalcolm** [comment]
Thanks - fixed
